src/screens/SendSheet.tsx

```diff
@@ -669,9 +672,14 @@ export default function SendSheet() {
       const goBackAndNavigate = () => {
         goBack();
         navigate(Routes.WALLET_SCREEN);
-        InteractionManager.runAfterInteractions(() => {
-          navigate(Routes.PROFILE_SCREEN);
-        });
+
+        // with toasts: just show the toast
+        // without toasts: navigate user to the activity list
+        if (!rainbowToastsEnabled) {
+          InteractionManager.runAfterInteractions(() => {
+            navigate(Routes.PROFILE_SCREEN);
```

Comment:
there are a couple other places where we trigger this navigation behavior after a transaction, outside of just the send sheet, e.g, NFTSingleOfferSheet/index.ts, MintSheet.tsx (we can ignore updating ENSConfirmRegisterSheet.tsx as we'll likely be ripping that out soon)